code/datums/traits/neutral.dm

```diff
@@ -0,0 +1,27 @@
+//traits with no real impact that can be taken freely
+//MAKE SURE THESE DO NOT MAJORLY IMPACT GAMEPLAY. those should be positive or negative traits.
+
+/datum/trait/no_taste
+	name = "Ageusia"
+	desc = "You can't taste anything! Toxic food will still poison you."
+	value = 0
+	mob_trait = TRAIT_AGEUSIA
+	gain_text = "<span class='notice'>You can't taste anything!</span>"
+	lose_text = "<span class='notice'>You can taste again!</span>"
+	medical_record_text = "Patient suffers from ageusia and is incapable of tasting food or reagents."
+
+
+
+/datum/trait/deviant_tastes
+	name = "Deviant Tastes"
+	desc = "You dislike food that most people enjoy, and find delicious what they don't."
+	value = 0
+	gain_text = "<span class='notice'>You start craving something that tastes strange.</span>"
+	lose_text = "<span class='notice'>You feel like eating normal food again.</span>"
+
+/datum/trait/deviant_tastes/add()
+	var/mob/living/carbon/human/H = trait_holder
+	var/datum/species/species = H.dna.species
+	var/liked = species.liked_food
+	species.liked_food = species.disliked_food
+	species.disliked_food = liked
```

Comment:
Isn't this missing a remove() proc?

Reply:
Where would I put the remove proc?

Reply:
On the /datum/trait/deviant_tastes/remove() ?

Reply:
Oh, right. I never intend for trait datums to be removed to begin with, but I do suppose it'd be good to be there in case they were.

Reply:
I mean for testing and admins messing around in the thunderdome.